### ISSUE TITLE
Githook should listen for main or master

### ISSUE THREAD

**Issue by fmendozaro** [zgulde/cods#35]
With the latest changes on github, our default branch is most likely to be `main`, but the cods githook on the server-side only triggers the deployment commands if it comes from `master`.

Right now the solution is just to create a `master` that mirrors `main`.

**Comment by zgulde** [comment]
I agree and should probably update the hook. In the meantime, here's a quick hacky fix:

```
git push production main:master
```

That will push the `main` branch to the production remote as `master`
